gandalf.js

```diff
@@ -4,32 +4,30 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!